client/app/main/dashboard/dashboard.controller.js

```diff
@@ -37,7 +126,8 @@ function DashboardCtrl($scope, $timeout, AbsenceRecord, GridDefaults, Student) {
         $scope.loading = false;
         $timeout($scope.gridApi.treeBase.expandAllRows);
       });
-      $scope.menuItems.length = 1;
+      $scope.menuItems.length = 0;
+      [].push.apply($scope.menuItems, defaultMenuItems);
```

Comment:
MDN explains it fairly well.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call